sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md

```diff
@@ -119,14 +135,28 @@ The component [Tower](https://github.com/kubesphere/tower) of KubeSphere is used
 
 ### 3.2. <a name='MemberCluster-Agent'></a>Install Member Cluster
 
-* There is no difference between the installation of Member Cluster and the installation of common clusters that have their multi-cluster feature disabled. Please make sure "multicluster" is set as below: 
+* In order for host cluster to manager member cluster, we need to make the jwtSecret same between them. So first get it from the host by the following command.
+
+    ```bash
+    $ kubectl -n kubesphere-system get cm kubesphere-config -o yaml | grep -v "apiVersion" | grep jwtSecret
+      jwtSecret: "gfIwilcc0WjNGKJ5DLeksf2JKfcLgTZU"
+    ```
+
+* There is no difference between the installation of Member Cluster and the installation of common clusters that have their multi-cluster feature disabled. Please make sure "multicluster" is set as the following, input the jwtSecret we have from the step above and wait for the installation to be completed.
```

Comment:
`input the jwtSecret we have from the step above` => `input the corresponding jwtSecret shown above`

Reply:
`as the following` is same as `as follows`. I am going to update the doc based on the second comment

Reply:
Ref: https://english.stackexchange.com/questions/68376/as-follows-vs-as-the-following

Reply:
Ok, I changed it. But `as the following` should appear everywhere in textbook.